test/videojs-flashls-source-handler.test.js

```diff
@@ -35,3 +36,62 @@ QUnit.module('videojs-flashls-source-handler', {
   }
 });
 
+QUnit.test('Flashls Audiotracks tests',
+  function(assert) {
```

Comment:
this should be able to fit on the same line as the `QUnit.test`, then the rest of this test can have one less level of indentation